### PR TITLE
WIP fix reference enum field

### PR DESCRIPTION
**Pull request by yishengjin1413**
This PR is used to fix wrong enum field reference.  

The fix is based on the old content from CAPS, so it may have some content are fixed by mistake. So please help to have a review.

### CODE REVIEW

File: xml/System.DirectoryServices.AccountManagement/PrincipalCollection.xml [hunk 1]
```diff
@@ -316,7 +316,7 @@
   
 |Exception|Description|  
 |---------------|-----------------|  
-|<xref:System.InvalidOperationException>|If this method is called on the <xref:System.DirectoryServices.AccountManagement.GroupPrincipal.Members> PrincipalCollection, the <xref:System.DirectoryServices.AccountManagement.GroupPrincipal.Members> property cannot be cleared because one or more of the group's members have this group as their primary group.|  
+|<xref:System.InvalidOperation>|If this method is called on the <xref:System.DirectoryServices.AccountManagement.GroupPrincipal.Members> PrincipalCollection, the <xref:System.DirectoryServices.AccountManagement.GroupPrincipal.Members> property cannot be cleared because one or more of the group's members have this group as their primary group.|
```
Comment (NextTurn) [inline xml/System.DirectoryServices.AccountManagement/PrincipalCollection.xml]: 😕

Comment (mairaw) [reply xml/System.DirectoryServices.AccountManagement/PrincipalCollection.xml]: I'll be reviewing this PR @NextTurn and only bringing the files with changes that we really want to bring.

**Comment by mairaw** [comment]
@yishengjin1413 where did this branch/repo go?

**Comment by yishengjin1413** [comment]
@mairaw Seems I can't find it either... But you can find all the files in this commit branch: https://github.com/dotnet/dotnet-api-docs/tree/6224b01403dd76f51479434576913356ca77389b

**Pull request closed without merging**